<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PickResultLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="text">
     <string>学号-姓名的点名器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowHistoryButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查看点过的名字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MulPickButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>连抽模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PickOneButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>单抽开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QListWidget" name="HistoryListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenNameFileButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>111</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>打开名字文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RePickButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>121</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>重新开始点名</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="MulPickTabWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>501</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QWidget" name="mulpick">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLineEdit" name="PeopleLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="NumberLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>258</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>连抽人数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StartMulButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QListWidget" name="MulHistoryListWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>461</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="PersonalInformationTabWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>491</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QWidget" name="personal">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>57</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>班级</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>xxx</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>图片</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
